### ISSUE TITLE
:raw-link Not working

### ISSUE THREAD

**Issue by floscr** [ndwarshuis/om.el#7]
If I got a headline 

```org
* TODO [[https://xyz.tld][FooBar]]
```

And try to get the raw-link via

```elisp
(->> (om-parse-this-headline)
     (om-get-property :title)
     (-first-item)
     (om-get-property :raw-link))
```

I get the error

`om--arg-error: Argument type error: "Unsettable property ’:raw-link’ for type ’link’ requested; settable properties are :path, :format, :type, :raw-link, :application, :search-option, :post-blank"`

**Comment by ndwarshuis** [comment]
should be fixed with f6de9fcdba10be1ce4901e05fb143408c5e7d2fe

**Comment by floscr** [comment]
Thank you, works now!
